File: Max, Min, Mean/max_min_mean.py
# Given an array, print the max, min and average values for that array.
def mmm(arr):
    arr_max = arr[0]
    arr_min = arr[0]
    total = arr[0]
    avg = 0
    for i in range(1, len(arr)):
        total += arr[i]
        if arr[i]>arr_max:
            arr_max = arr[i]
        if arr[i]<arr_min:
            arr_min = arr[i]
    avg = total/len(arr)
    print(f"max {arr_max}, min {arr_min}, avg {avg}")
    return arr_max, arr_min, avg

mmm([-5,-5,3,4,1,2,5,5])

File: Max, Min, Mean/max_min_mean.js
// Given an array, print the max, min and average values for that array.
function mmm(arr){
    let max = arr[0];
    let min = arr[0];
    let total = arr[0];
    let avg = 0;
    for(i=1; i<arr.length; i++){
        total += arr[i];
        if(arr[i]>max){
            max = arr[i];
        }
        if(arr[i]<min){
            min = arr[i];
        }
    }
    avg = total/arr.length
    console.log("max", max, "min", min, "avg", avg);
    return max, min, avg;
}

mmm([-5,-5,3,4,1,2,5,5])